File: practice_4-26/Prac.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class Prac {
    public static void main(String[] args) {
        String note = "He old";
        String magazine = "Hello World";
        System.out.println(canNoteBeMade(note.replaceAll(" ", ""), magazine.replaceAll(" ", "")));
    }

    public static boolean canNoteBeMade(String note, String magazine) {
        List<String> noteList = new ArrayList<String>(Arrays.asList(note.split("")));
        List<String> magList = new ArrayList<String>(Arrays.asList(magazine.split("")));
        // for (String c : noteList) {
        // if (magList.contains(c)) {
        // magList.remove(c);
        // } else {
        // return false;
        // }
        // }
        int i = 0;
        while (i < noteList.size()) {
            if (magList.contains(noteList.get(i))) {
                magList.remove(noteList.get(i));
            } else {
                return false;
            }
            i++;
        }
        return true;
    }
}

File: practice_4-26/prac.py
def can_make_note(note, mag):
    note = list(note.replace(" ", ""))
    mag = list(mag.replace(" ", ""))
    for c in note:
        if c in mag:
            mag.remove(c)
        else:
            return False
    return True


def main():
    note = "He old"
    mag = "Hllo World"
    print(str(can_make_note(note, mag)))


if __name__ == "__main__":
    main()
